modules/bridge/src/main/ts/ephox/bridge/components/dialog/Slider.ts

```diff
@@ -0,0 +1,25 @@
+import { FieldSchema, StructureSchema } from '@ephox/boulder';
+
+import { formComponentFields, FormComponent, FormComponentSpec } from './FormComponent';
+
+export interface SliderSpec extends FormComponentSpec {
+  type: 'slider';
+  label: string;
+  min?: number;
+  max?: number;
+}
+
+export interface Slider extends FormComponent {
+  type: 'slider';
+  label: string;
+  min: number;
+  max: number;
+}
+
+export const sliderSchema = StructureSchema.objOf(
```

Comment:
This is what I meant when I said I was quite rusty on Alloy and probably needed help getting it right :)

Reply:
Yeah I had forgotten about it myself tbh. I only remembered when I saw your other changes to the panel schema.

Reply:
looking at `sizeinput`, I don't think we need a data processor for `min` and `max` as they're part of the spec. It's the value that will be part of the data and it's just a simple number?

Reply:
That's not how the data processor works though... if you look at https://github.com/tinymce/tinymce/blob/8ee937c891eabef49cb86cce605680b913915346/modules/bridge/src/main/ts/ephox/bridge/core/DialogData.ts#L7 if it can't find a registered data processor then it just ignores the data because a `FieldSchema` isn't set up.

Take a look at https://github.com/tinymce/tinymce/blob/8ee937c891eabef49cb86cce605680b913915346/modules/bridge/src/main/ts/ephox/bridge/components/dialog/Textarea.ts#L28 and you'll see we still need something there even if it's a primitive data type.

Reply:
Yeah I went with `export const sliderInputDataProcessor = ValueType.number;` (and then as discussed elsewhere this doesn't work due to the Alloy component using an object)